airgun/views/configgroup.py

```diff
@@ -18,6 +19,7 @@ class ConfigGroupsView(BaseLoggedInView, SearchableViewMixin):
             'Actions': Text('.//a[@data-method="delete"]'),
         },
     )
+    dialog = Pf4ConfirmationDialog()
```

Comment:
@vsedmik Pf4ConfirmationDialog is common in multiple components so we moved it to https://github.com/SatelliteQE/airgun/blob/a2a8be1670196b11ad1c577fa7081c53c10bdde8/airgun/views/common.py#L44